extensions/entsoe/include/powsybl/iidm/extensions/entsoe/MergedXnodeAdder.hpp

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_EXTENSIONS_ENTSOE_MERGEDXNODEADDER_HPP
+#define POWSYBL_IIDM_EXTENSIONS_ENTSOE_MERGEDXNODEADDER_HPP
+
+#include <string>
+
+#include <powsybl/iidm/ExtensionAdder.hpp>
+#include <powsybl/stdcxx/math.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+namespace extensions {
+
+namespace entsoe {
+
+class MergedXnodeAdder : public ExtensionAdder {
+public:
+    /**
+     * Constructor
+     */
+    explicit MergedXnodeAdder(Extendable& extendable);
+
+    /**
+     * Destructor
+     */
+    ~MergedXnodeAdder() noexcept override = default;
+
+    /**
+     * Copy constructor
+     */
+    MergedXnodeAdder(const MergedXnodeAdder& extendable) = default;
```

Comment:
Remove this parameter's name like in the other extension

Reply:
Done